jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/interfaces/asserts/MessagesAssert.java

```diff
@@ -0,0 +1,51 @@
+package com.epam.jdi.light.vuetify.interfaces.asserts;
+
+import com.epam.jdi.light.common.JDIAction;
+import com.epam.jdi.light.vuetify.interfaces.HasMessages;
+import org.hamcrest.Matchers;
+
+import java.util.List;
+
+import static com.epam.jdi.light.asserts.core.SoftAssert.jdiAssert;
+
+public interface MessagesAssert<A, E extends HasMessages> extends VuetifyAssert<E> {
+
```

Comment:
Maybe we also can assert that there are no error messages. To check that everything is going fine

Reply:
added